tests/phpunit/integration/Modules/AdsTest.php

```diff
@@ -11,56 +11,181 @@
 namespace Google\Site_Kit\Tests\Modules;
 
 use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
 use Google\Site_Kit\Core\Tags\GTag;
 use Google\Site_Kit\Modules\Ads;
+use Google\Site_Kit\Modules\Ads\Settings;
+use Google\Site_Kit\Tests\Fake_Site_Connection_Trait;
 use Google\Site_Kit\Tests\TestCase;
+use WP_User;
 
 /**
  * @group Modules
  * @group Ads
  */
 class AdsTest extends TestCase {
 
+	private $user_id;
+
+	/**
+	 * Ads object.
+	 *
+	 * @var Ads
+	 */
+	private $ads;
+
+	/**
+	 * @var User_Options
+	 */
+	protected $user_options;
+
+	/**
+	 * @var Options
+	 */
+	protected $options;
+
+	/**
+	 * @var Authentication
+	 */
+	protected $authentication;
+	use Fake_Site_Connection_Trait;
```

Comment:
This trait is added in a strange place here – let's move it up to the top of the class body, before all the properties.

Reply:
This is not needed actually, it was leftover by automatic import, not sure why it placed it in weird place, so I didn't notice it in previous edit.